reason-react-native/src/apis/LayoutAnimation.md

```diff
@@ -5,6 +5,11 @@ title: LayoutAnimation
 
 `LayoutAnimation` API offers a simpler alternative to `Animated` API. Instead of directly manipulating values for various style props, it suffices to specify the animation to be run before the next render. Specification of the animation should happen in the reducer, before `state` is updated.
 
+`LayoutAnimation` is still experimental on Android and needs to be explicitly enabled with the `UIManager` method `setLayoutAnimationEnabledExperimental`. However, as that method should be removed when `LayoutAnimation` is no longer experimental, the external declaration for it wraps its type in `option` to avoid runtime errors when that happens. `LayoutAnimation` may be enabled, if it is not already enabled by default, by means of a statement such as:
+```reason
+Belt.Option.mapWithDefault(UIManager.setLayoutAnimationEnabledExperimental, (), f => true -> f);
```

Comment:
Maybe writing it out like this would be clearer for beginners (although longer):

```re
switch (UIManager.setLayoutAnimationEnabledExperimental) {
| Some(setEnabled) => setEnabled(true)
| None => ()
};
```
?

Reply:
What about

```reason
UIManager.setLayoutAnimationEnabledExperimental
->Belt.Option.map(setEnabled => setEnabled(true))
->ignore;
```

Reply:
I decided to keep it simple and went with the `switch` statement. `ignore` may need explanation of itself.